mkdocs.yml

```diff
@@ -11,14 +11,16 @@ copyright: Copyright &copy; 2018 - 2024 TrenchBoot Developers
 
 nav:
     - Home: index.md
-    - Events:
+    - Events and publications:
       - events.md
       - upcoming_events.md
+      - publications.md
     - Specifications:
       - specifications/index.md
       - specifications/Secure_Launch.md
     - Documentation:
       - Getting started:
+        - user-docs/QUICKSTART.md
         - user-docs/index.md
         - user-docs/requirements.md
         # - installing - Linux
```

Comment:
This is a placeholder for `QUICKSTART.md`, move it here. Index must be the first item on the list, and having requirements before installation instructions may save some frustrations, assuming users will go through the menu in order.

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/pull/33/commits/b118d28d84f5942ed52a4177fd051642c23d367c

Reply:
Did you mean installation instructions before requirements? I don't get why the other way around would be frustrating. You first get the requirements and then try to install.

Reply:
No, what you did is what I meant.